module-2/lab-matplotlib-seaborn/your-code/challenge-3.ipynb

```diff
@@ -9,7 +9,7 @@
   },
```

Comment:
- Regarding this question: "Whether the bin size affects your plot? Why?The bin size affects the plot because the more bins the more segregated the data is.In the Histogram with 50 bins it's tougher to find a pattern than in the histogram with 10 bins".  
There are a few general rules for choosing bins:

1. Bins should be all the same size. For example, groups of ten or a hundred.
2. Bins should include all of the data, even outliers. If your outliers fall way outside of your other data, consider lumping them in with your first or last bin. This creates a rough histogram —make sure you note where outliers are being included.
3. Boundaries for bins should land at whole numbers whenever possible (this makes the chart easier to read).
Choose between 5 and 20 bins. The larger the data set, the more likely you’ll want a large number of bins. For example, a set of 12 data pieces might warrant 5 bins but a set of 1000 numbers will probably be more useful with 20 bins. The exact number of bins is usually a judgment call.
4. If at all possible, try to make your data set evenly divisible by the number of bins. For example, if you have 10 pieces of data, work with 5 bins instead of 6 or 7.
- Regarding this question: "Use the appropriate plot to visualize the column Gender. There are 2 ways of doing it. Do it both ways". You can also use the seaborn countplot. 
```Python
sns.countplot()
```
- Regarding this "Age is skewed since it has a bunch of "older" passengers", it has very little skewness. The Age distribution is nearly symmetrical , we can't think in a skewed distribution. You can see both in the violin plot, the histogram and the boxplot. 

- Try this the other way around, plotting age in y axis and Fare in x axis. 
```Python
x = titanic.Age
y = titanic.Fare
```